modules/claims_api/spec/requests/v1/claims_request_spec.rb

```diff
@@ -44,6 +44,7 @@
   before do
     stub_poa_verification
     stub_mpi
+    stub_jwt_valid_token_decode
```

Comment:
I'm not sure why this was added for all of these files but it's certainly not necessary. These changes are unrelated to these files

Reply:
I think he needs to rebase and drop some commits. It looks like he merged.

Reply:
Yeah I don't know what happened looks like I added commits from someone else on accident. I will just create a new branch and a new pull request.